manifest.samples.json

```diff
@@ -1,8 +1,9 @@
 {
+  "registry": "mcr.microsoft.com",
```

Comment:
Should aspnetmvcapp be listed here as well?

Reply:
IDK - possibly.  This is outside the scope of the MCR migration.  Will log a separate issue to track this.

Reply:
https://github.com/Microsoft/dotnet-framework-docker/issues/238